demo/basel.R

```diff
@@ -351,7 +351,7 @@ dm_with_summary %>%
 dm_flights %>%
   dm_zoom_to_tbl(flights) %>%
   count(carrier, origin) %>%
-  dm_insert_zoomed_tbl(origin_carrier_count) %>%
+  dm_insert_zoomed(origin_carrier_count) %>%
```

Comment:
```suggestion
  dm_insert_zoomed("origin_carrier_count") %>%
```

New objects should be quoted by the caller. Maybe add a warning in `dm_insert_zoomed()` (in a new issue/PR)?